modules/PhyVersoBSP/phyverso_mcu_comms/evSerial.cpp

```diff
@@ -209,11 +209,11 @@ bool evSerial::handle_OpaqueData(const uint8_t* buf, const int len) {
     // Lambda for updating OpaqueDataHandler - here just to simplify the return
     // logic.
     [this](const OpaqueData& data) {
-        auto iter = psensor_handlers.find(data.connector);
-        if (iter == psensor_handlers.end()) {
+        auto iter = opaque_handlers.find(data.connector);
+        if (iter == opaque_handlers.end()) {
```

Comment:
Nit:
This can be simplified to emplace always and use the returned boolean to check if element was added to the map or not
```
const auto [iter, inserted] = opaque_handlers.emplace(data.connector, data);
if (not inserted) {
   iter->second.insert(data);
}
```

Reply:
this is more readable but it would result in a second find below

Reply:
What will be the second find and for what reason?